deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1372,7 +1372,7 @@ spec:
                   - name: OPERATOR_TAG
                     value: v0.1.10
                   - name: WEBUI_TAG
-                    value: v0.1.10
+                    value: v2.0.0-14.8
```

Comment:
This looks like a d/s value. Is it?

Reply:
No, the whole point about this is that upstream web-ui releases are tagged as downstream while web-ui-operator is not.
See:
https://github.com/kubevirt/web-ui/releases
https://quay.io/repository/kubevirt/kubevirt-web-ui?tab=tags

Reply:
I follow now.